.github/workflows/sdfactorypipeline.yml

```diff
@@ -44,7 +22,7 @@ env:
   # Use GH Container Registry instead of docker
   # github.repository equals to <account|org>/<repo>
   REGISTRY: ghcr.io
-  IMAGE_NAME: catenax-ng/tx-sd-factory/sdfactory
+  IMAGE_NAME: cofinity-x/cofinity-x-sd-factory-core/sdfactory
```

Comment:
This is a no-go! This is a Eclipse-TractusX repository and the build artifacts should not be pushed to a container registry which belongs obviously to a commercial organization. I see big legal issues here!